google/cloud/storage/oauth2/service_account_credentials.h

```diff
@@ -37,6 +39,10 @@ struct ServiceAccountCredentialsInfo {
   std::string private_key_id;
   std::string private_key;
   std::string token_uri;
+  // Optional; if empty, a default set of scopes will be used.
```

Comment:
The comment says it's "optional", so perhaps the type should be `optional<set<string>>`?